frontend/packages/client/src/components/common/Toast.js

```diff
@@ -1,43 +1,38 @@
-import { useHistory } from 'react-router-dom';
 import { Svg } from '@cast/shared-components';
 
-const Toast = ({ onClose, text, footerText, footerLink }) => {
-  const history = useHistory();
-
-  return (
+const Toast = ({ history, onClose, text, footerText, footerLink }) => (
```

Comment:
so this definitely worked before where I was able to call `useHistory` inside the toast component but it suddenly started becoming undefined. Not really sure why that stopped working but calling `useHistory` in the toast hook seemed logical and makes it work in all cases so updated to that method